### PR TITLE
Add HEVC registration

### PR DESCRIPTION
**Pull request by sandersdan**

### CODE REVIEW

File: hevc_codec_registration.src.html [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    "date": "September 2019"
+  }
+}
+</pre>
+
+Fully qualified codec strings {#fully-qualified-codec-strings}
+==============================================================
+
+The codec string begins with the prefix "hev1." or "hvc1.", with a suffix of
+four dot-separated fields as described in Section E.3 of [[iso14496-15]].
```
Comment (sandersdan) [inline hevc_codec_registration.src.html:75]: I did not include hvc2, hev2, lhv1, lhe1, which are specific to L-HEVC and perhaps will be added at a later date.

File: hevc_codec_registration.src.html [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+  </dd>
+</dl>
+
+HevcBitstreamFormat {#hevc-bitstream-format}
+------------------------------------------
+<pre class='idl'>
+<xmp>
+enum HevcBitstreamFormat {
+  "annexb",
+  "hevc",
```
Comment (sandersdan) [inline hevc_codec_registration.src.html:177]: I'm not sure if there is a common name for this. For H.264 I think it was commonly called "AVC format" leading to the format string "avc", but perhaps this one should be called "bmff" which would be more precise.

File: hevc_codec_registration.src.html (hunk 1 above)
Comment (aboba) [reply hevc_codec_registration.src.html:75]: Title: line should use H.265, not H.264.

File: hevc_codec_registration.src.html (hunk 2 above)
Comment (aboba) [reply hevc_codec_registration.src.html:177]: "hevc" might be more familiar.

File: hevc_codec_registration.src.html [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+    start from the middle of the stream.
+
+EncodedVideoChunk type {#encodedvideochunk-type}
+================================================
+
+If an {{EncodedVideoChunk}}'s [=EncodedVideoChunk/[[type]]=] is
+{{EncodedVideoChunkType/key}}, and the bitstream is in
+{{HevcBitstreamFormat/hevc}} format, then the {{EncodedVideoChunk}} is expected
+to contain a base layer primary coded picture that is an instantaneous decoding
+refresh (IDR), clean random access (CRA), or broken link access (BLA) picture.
```
Comment (sandersdan) [inline hevc_codec_registration.src.html:122]: I wrote this to match the definition of "sync sample" in the ISO BMFF embedding, but I'm not familiar enough with decoder implementations to say for sure if that's the best definition.

**Comment by chrisn** [comment]
Please let me know when this is ready, and I can run a call for consensus in the Media Working Group.

**Comment by chcunningham** [comment]
> Please let me know when this is ready, and I can run a call for consensus in the Media Working Group.

Please go ahead and run the CFC.

File: hevc_codec_registration.src.html [hunk 4]
```diff
@@ -0,0 +1,212 @@
+<pre class='metadata'>
+Title: HEVC (H.265) WebCodecs Registration
+Repository: w3c/webcodecs
+Status: NOTE-ED
+Shortname: webcodecs-avc-codec-registration
```
Comment (chrisn) [inline hevc_codec_registration.src.html:5]:
```suggestion
Shortname: webcodecs-hevc-codec-registration
```

**Review by aboba — APPROVED** [review]

**Comment by zeruniverse** [comment]
I saw CfC closed with no objection. Will it gets merged? https://lists.w3.org/Archives/Public/public-media-wg/2022Jul/0008.html

**Comment by chcunningham** [comment]
Yes, plan is to merge. I'm taking a closer look at the PR now.

**Comment by chrisn** [comment]
I think the only outstanding thing is to seek review as @aboba suggested in https://github.com/w3c/webcodecs/pull/499#issuecomment-1170070172. That may not need to hold up merging though.

File: hevc_codec_registration.src.html (hunk 3 above)
Comment (chcunningham) [reply hevc_codec_registration.src.html:122]: Also not familiar enough. Maybe Gary could weigh here.

File: hevc_codec_registration.src.html (hunk 2 above)
Comment (chcunningham) [reply hevc_codec_registration.src.html:177]: +1 for hevc, seems less confusing.

**Review by chcunningham — APPROVED** [review]
LGTM. I only found one minor editorial issue. Dan is OOO, so I'll go ahead and merge this and fix the editorial glitch in a follow up commit.

File: hevc_codec_registration.src.html [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+    type: interface
+        text: EncodedVideoChunk; url: encodedvideochunk
+        text: VideoEncoder; url: videoencoder
+    type: dictionary
+        text: VideoEncoderConfig; url: dictdef-videoencoderconfig
+        text: VideoDecoderConfig; url: dictdef-videodecoderconfig
+</pre>
+
+<pre class='biblio'>
+{
```
[the remark this answers is not in the document]
Comment (chcunningham) [reply hevc_codec_registration.src.html:55]: Done. https://github.com/w3c/webcodecs/commit/72baacb43c74632653c9cc3632f3e10dd274fac9

[the remark this answers is not in the document]
Comment (sandersdan) [reply hevc_codec_registration.src.html:55]: Not quite true, H.265 also has an Annex B. It's basically identical to H.264 Annex B.

**Pull request merged**